packages/client3/components/design-system/typography/body-text.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from "clsx";
+import { HTMLAttributes } from "react";
+
+export interface BodyTextProps extends HTMLAttributes<HTMLHeadingElement> {
+  size: "large" | "medium" | "normal" | "small";
+  semiBold?: boolean;
+}
+
+export function BodyText({
+  size,
+  semiBold = false,
+  className,
+  ...rest
+}: BodyTextProps) {
+  const Component = "p";
```

Comment:
Just asking, is there any reasoning for declaring the component at the top?

Reply:
fixed